### ISSUE TITLE
Broken item list display

### ISSUE THREAD

**Issue by cNori** [FlaxEngine/FlaxEngine#2609]
**Issue description:**
![image](https://github.com/FlaxEngine/FlaxEngine/assets/162097313/206d2e9e-acd4-4c6b-8de6-ec302cbc763c)

**Steps to reproduce:**
create script add a filed 
```c#
    public struct Potato
    {
        public enum PoatatoType
        {
            none,
            rusted
        }
        public int LestTestThis;
    }
    public List<Potato> m_PotatoList;
```

**Flax version:**
https://github.com/FlaxEngine/FlaxEngine/commit/6a883bc7c6d5396ef8061e018d8a2ee8e0fe589d

**Comment by Menotdan** [comment]
Does it still break if you remove the enum declaration? The enum declaration seems out of place.

**Comment by cNori** [comment]
> Does it still break if you remove the enum declaration? The enum declaration seems out of place.

its the same

**Comment by Tryibion** [comment]
This and <https://github.com/FlaxEngine/FlaxEngine/issues/2530> stem from the same issue. There is something not working with when it tries to reuse a properties list. It is like the control child index that the labels tries to use is off/does not get assigned correctly.

**Comment by mafiesto4** [comment]
Fixed in 68eb8b9f212db01edde27d1805abefd53df2100f.
